doc/api/esm.md

```diff
@@ -591,15 +591,17 @@ would provide the exports interface for the instantiation of `module.wasm`.
 
 ## Top-level `await`
 
+<!--
+added: v14.8.0
+-->
+
 > Stability: 1 - Experimental
 
 The `await` keyword may be used in the top level (outside of async functions)
```

Comment:
Perhaps hyphenate top-level? Also perhaps move `top-level body` together before the `(outside of async functions)` rather than splitting it up?

Reply:
The wording in the spec is

> Top-Level Await allows the `await` keyword to be used at the top level of the module goal.

It looks to me that we should keep it without hyphen. Agreed we should move "body" before the parenthesis though 👍

Reply:
Maybe we can remove the parenthesis altogether, I don't think bringing async function helps here.